kansha/board/comp.py

```diff
@@ -847,6 +857,22 @@ def search(self, query):
                 self.card_matches.add(None)
         else:
             self.card_matches = set()
+            
+    @classmethod
```

Comment:
These method can't be classmethods because they don't return Boards instance but DataBoards instance. So, they should either be classmethods of DataBoard or static methods of Board.